Fundamentals/Setup/Install/install-umbraco-with-vs-code.md

```diff
@@ -59,15 +59,15 @@ With that, you're ready to run! Press F5, or click the little green play button
 
 This section continues from where we left off but covers the installation and configuration of Umbraco inside your web browser when you run Umbraco for the first time.
 
-1. You will see the install screen. If you've installed the template with `-ce` flag, to enable SqlCE, you only need to fill out your information and click "Install", the installer will take care of the rest. If you've enabled SqlCE, but still want to change the database type, click "Customize". If you didn't enable SqlCE, you will see a "Next" button instead of "Install", which will take you to an extra step where you can configure your database.
+You will see the install screen where you will need to fill in some data before Umbraco can be installed.
 
-    ![Web Installer - Lets Get Started](images/VsCode/installer-v9.png)
+If you've installed the template with `-ce` flag, to enable SqlCE, you only need to fill out your information and click "Install", the installer will take care of the rest. If you've enabled SqlCE, but still want to change the database type, click "Change Database". If you didn't enable SqlCE, you will see a "Next" button instead of "Install", which will take you to an extra step where you can configure your database.
```

Comment:
This is a very big para with multiple options and instructions. Can we split each action/instruction into a point for ease of readability maybe something like:

2. **If you've installed the template with `-ce` flag** - To enable SqlCE, you only need to fill out your information and click **Install**, the installer will take care of the rest.
3. **If you've enabled SqlCE but still want to change the database type** -  Click **Change Database**.
4. **If you didn't enable SqlCE** - You will see a **Next** button instead of **Install** which will take you to an extra step where you can configure your database.

Reply:
I agree that this part could do with a rewrite.
I honestly just left most of what was already in the article 😅 
I'll make some changes, to make it a little easier to understand the various options! 💪